src/io/flutter/preview/WidgetEditToolbar.java

```diff
@@ -0,0 +1,396 @@
+/*
+ * Copyright 2019 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.preview;
+
+import com.google.common.util.concurrent.Uninterruptibles;
```

Comment:
This is a simple refactor of the toolbar code from the PreviewView into its own class.
No functionality was added or changed. The one difference is EventStream objects are used to ensure this class is reliably notified on changes to the selected outline and selected file.